client/src/reducers/index.js

```diff
@@ -33,6 +38,16 @@ function rootReducer(state = initialState, action) {
         ...state,
         user: action.payload,
       };
+    case SET_ERROR:
```

Comment:
I'm wondering if there's a way to prevent having to store all the errors in the global state if we don't needed.
This is kind of "the Redux way" but I'm also working on the login / signup form error handling at the moment... and I'm thinking it's not very useful to keep this in the global state if we don't need to persist the errors.
Also, it could lead to bugs and sharing errors from different components here with the same state. At least we could maybe split up the state for multiple components... some examples are listed here in this article too https://dispatch.moonfarmer.com/redux-status-and-error-methodology-ec5c1f1634b7